src/test-utils/service-mocks.ts

```diff
@@ -180,6 +179,13 @@ export class MockAuthService {
   validateApiKey = jest.fn();
 }
 
+export class MockTrustedClaimIssuersService {
```

Comment:
Prashant added this package recently: https://www.npmjs.com/package/@golevelup/ts-jest

You should be able to do `createMock<MockTrustedClaimIssuersService>()` where ever you need this instead.

You might want to define a Nest `Provder` object in this file to save on test boiler plate.